support/bootstrap/master/ubuntu/master1.sh

```diff
@@ -19,6 +19,8 @@ package="/twindb-backup/omnibus/pkg/twindb-backup_${TB_VERSION}-1_amd64.deb"
 
 apt -y install "$package"
 
+chmod 644 /etc/mysql/mysql.conf.d/mysqld.cnf
```

Comment:
jammy docker image has this file with o777 permissions which broke integration tests. MySQL refuses to read a config file if it's others writable.